docs/deployment/application-deployment.md

```diff
@@ -1,77 +1,77 @@
 # Deploying to Dokku
 
-> Note: This document uses the hostname `dokku.me` in commands. For your server, please
-> substitute your server's hostname instead.
+**Note:** This walkthrough uses the hostname `dokku.me` in commands. When deploying to your own server, you should set the `DOKKU_DOMAIN` value in the `Vagrantfile` before you initialize the Vagrant VM.
 
 ## Deploy tutorial
 
-Once Dokku has been configured with at least one user, applications can be deployed via a `git push` command. To quickly see Dokku deployment in action, you can use the Heroku Ruby on Rails example app.
+Once you have configured Dokku with at least one user, you can deploy applications using `git push`. To quickly see Dokku deployment in action, try using [the Heroku Ruby on Rails "Getting Started" app](https://github.com/heroku/ruby-getting-started).
 
 ```shell
-# from your local machine
+# From your local machine
 # SSH access to github must be enabled on this host
-git clone git@github.com:heroku/ruby-rails-sample.git
+git clone git@github.com:heroku/ruby-getting-started.git
 ```
 
 ### Create the app
 
-Create the application on the Dokku host. You will need to SSH onto the host to run this command.
+SSH into the Dokku host and create the application as follows:
 
 ```shell
-# on the Dokku host
-dokku apps:create ruby-rails-sample
+# On the Dokku host
+dokku apps:rails-sample-app
 ```
 
 ### Create the backing services
 
-When you create a new app, Dokku by default *does not* provide any datastores such as MySQL or PostgreSQL. You will need to install plugins to handle that, but fortunately [Dokku has official plugins](/docs/community/plugins.md#official-plugins-beta) for common datastores. Our sample app requires a PostgreSQL service:
+Dokku by default **does not** provide datastores (e.g. MySQL, PostgreSQL) on a newly created app. You can add database support by installing plugins, and [Dokku has official plugins](/docs/community/plugins.md#official-plugins-beta) for common datastores.
+
+The Getting Started app requires a PostgreSQL service, so install the plugin and create the related service as follows:
 
 ```shell
-# on the Dokku host
-# install the postgres plugin
-# plugin installation requires root, hence the user change
+# On the Dokku host
+# Install the postgres plugin
+# Plugin installation requires root, hence the user change
```

Comment:
For code comments, the style in the documentation is not to use uppercase for the first character, so this should be reverted everywhere.